server.js

```diff
@@ -3636,6 +3674,11 @@ function fetchFromSvg(request, url, cb) {
   });
 }
 
+function ordinalNumber(n) {
+  var s=["ᵗʰ","ˢᵗ","ⁿᵈ","ʳᵈ"], v=n%100;
```

Comment:
I can't think of anything; I fear we are stuck with that choice. If you consider this to look better, I'll merge this.